src/DatePicker/DatePicker.test.js

```diff
@@ -268,6 +292,31 @@ describe('<DatePicker />', () => {
         });
     });
 
+    describe('onChange callback', () => {
+        test('should call onChange on focusing input', () => {
+            const change = jest.fn();
+            const element = mount(<DatePicker defaultValue='2020-03-13' onChange={change} />);
+
+            element
+                .find('input[type="text"]')
+                .simulate('change', { target: { value: '04/14/2020' } });
+
+            expect(change).toHaveBeenCalledWith(expect.objectContaining({ formattedDate: '04/14/2020' }));
+        });
+    });
+
+    describe('onFocus callback', () => {
+        test('should call onFocus on entering input text', () => {
```

Comment:
Was the wording meant to be switched between these? i.e. 'should call onChange on entering input text', 'should call onFocus on focusing input'.

Reply:
Good catch, I swapped these around at some point.